### PR TITLE
Feature/unordered relations masked

### PR DESCRIPTION
**Pull request by mikeger**
### Issues

Fixed tests for unordered messages.

### CODE REVIEW

File: Sources/Helpers/ZMMessage+Dependency.swift [hunk 1]
```diff
@@ -90,7 +90,7 @@ extension ZMMessage {
         // so we iterate backwards and we ignore everything until we find this one
         var selfMessageFound = false
 
-        conversation.messages
+        NSOrderedSet(array: conversation.recentMessages)
```
Comment (vytis) [inline Sources/Helpers/ZMMessage+Dependency.swift]: Doesn't `enumerateObjects` work on array as well? Maybe we don't need to create a set here?

Comment (typfel) [reply Sources/Helpers/ZMMessage+Dependency.swift]: Bump

Comment (typfel) [reply Sources/Helpers/ZMMessage+Dependency.swift]: Couldn't you do normal swift enumeration and call `break` when you find your message.

Comment (mikeger) [reply Sources/Helpers/ZMMessage+Dependency.swift]: Great idea, updated it according to it!

**Comment by typfel** [comment]
@zenkins release major

**Pull request merged**